sdk/web-pubsub/web-pubsub/src/groupClient.ts

```diff
@@ -46,16 +46,19 @@ export interface GroupSendToAllOptions extends OperationOptions {
    * Connection ids to exclude from receiving this message.
    */
   excludedConnections?: string[];
+  /**
+   * The filter syntax to filter out the connections to send the messages to following OData filter syntax.
+   * Examples:
+   *  * Exclude connections from `user1` and `user2`: `userId ne 'user1' and userId ne 'user2'`
+   *  * Exclude connections in `group1`: `not('group1' in groups)`
+   */
+  filter?: string;
```

Comment:
since this is an OData filter, you might want to consider adding a template literal helper for escaping like we did in Search: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/search/search-documents/README.md#querying-with-odata-filters

Code lives here:
https://github.com/Azure/azure-sdk-for-js/blob/d7061ba7a583a0bcd0be2d398c7494525e773d54/sdk/search/search-documents/src/odata.ts#L38

Maybe it's not as useful if you don't accept strings that might have a quote character to escape in them though?

I'm also fine shipping this as-is and we can add more convenience/helpers later if devs have trouble assembling proper filter strings. Maybe @bterlson feels otherwise though?

Reply:
Nice suggestions! And nice doc for Search, I am going to add a similar one 😆 . Thanks for pointing me to the doc!

So the helper is to escape the values with `'` right? It could be useful when userId or group contains `'`, it might be rare case, but it sounds nice to add.